clients/python/build/README.md

```diff
@@ -0,0 +1,160 @@
+# Unity Catalog Python SDK
```

Comment:
since we are using the term "client" can we incorporate client in the names "Python Client SDK" for eg